src/licensedcode/data/licenses/proconx-modbus-rev4.LICENSE

```diff
@@ -5,7 +5,7 @@ name: proconX Modbus Driver License rev4
 category: Proprietary Free
 owner: proconX
 homepage_url: https://www.modbusdriver.com/info/LICENSE-FREE
-spdx_license_key: LicenseRef-scancode-proconX-modbus-rev4
```

Comment:
Let's add this as an other key with the incorrect case. And let's add two tests of an SPDX-License-Identifier using each the different case.

Reply:
I'm not sure we can add other SPDX license keys where only difference is character case, trying to add this failed with the following on reindex:
```
ValueError: Duplicated "other" SPDX license key: 'licenseref-scancode-proconx-modbus-rev4' defined in 'proconx-modbus-rev4' and License(key='proconx-modbus-rev4')
```
This is because we convert all of the keys to lowercase with this flag: https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/cache.py#L309 when we get a license_by_spdx_keys mapping in the index.